### ISSUE TITLE
Project website modv.vcync.gl doesn't work

### ISSUE THREAD

**Issue by illtellyoulater** [vcync/modV#938]
HTTP ERROR 503

**Comment by 2xAA** [comment]
I couldn't afford the domain renewal.

**Comment by illtellyoulater** [comment]
What about using `modv.github.io` ? It's freely provided by github.

**Comment by bennoH2025** [comment]
Create a free page on Wix.com or fix it, it can't be like that. At the very least you should remove the broken link everywhere, it's annoying.
Or just do it like someone says with ".io", but that can't be it here HTTP 503 error!

Erstellen Sie kostenlos eine Seite auf Wix.com oder Reparien Sie es, so kann das ja nicht sein. Zumindest solltet Ihr den nicht funktionierenden Link überall entfernen, das nervt.
Oder macht es halt wie da jemand sagt mit ".io", aber dass hier kan es wohl nicht sein HTTP 503 error!

**Comment by 2xAA** [comment]
https://modv.vcync.io/ is now live. If the domain lapses again, I will move to github.io

**Comment by illtellyoulater** [comment]
It's down again with `HTTP ERROR 503`

https://modv.vcync.gl/

please reopen the issue so others can help come up with a more stable solution

**Comment by jf-uu** [comment]
Isn't the domain now https://modv.vcync.io/? I don't think vcync owns vcync.gl anymore.

**Comment by illtellyoulater** [comment]
@jf-uu and others. Yes my apologies. I launched modV and clicked the link in the help menu which still points to the old domain, so I got confused. Please ignore this. It may still be required to address the wrong menu link, if it hasn't been done yet.

**Comment by jackwilsdon** [comment]
Yeah looks like https://modv.js.org (used in the help menu) still links to the old domain - I've opened #947 to change it.

**Comment by 2xAA** [comment]
Thanks for reporting. Have merged @jackwilsdon's PR.
